app/models/edition.rb

```diff
@@ -185,13 +185,20 @@ def substitute
 
   def gone?
     if state == "unpublished"
-      unpublishing = Unpublishing.find_by!(edition: self)
-      unpublishing.type == "gone"
+      unpublishing.gone?
     else
       document_type == "gone"
     end
```

Comment:
if you fancied a quick round of code golf, you could replace the method body with

`(state == "unpublished" && unpublishing.gone?) || document_type == "gone"`